### DISCUSSION TITLE
Cannot make it work again, after disabling the Win10 mode :( Please help

### DISCUSSION THREAD

**Discussion by bjoc** [valinet/ExplorerPatcher#225] [Q&A]
What a great project! 
Many MANY thanks for the developers!
Everthing worked until I disabled "enable windows 10" mode.
Now my new 11 taskbar shows up again and I cannot delete the dxgi.dll from Windows folder, even with admin rights, since Windows says, it is in use.
I cannot make ExplorerPatcher work again :(
Please help

**Comment by valinet** [comment]
Please read the uninstallation section of the README file.

Thanks

**Reply by Mathariman** [reply]
Hmm... Seems I am unable to even rename the file.  I have attempted to rename it with WSL while I have Explorer terminated, but Windows will still not allow it.

**Comment by bjoc** [comment]
Solution:
Since uninstallation does not touch the registry entries (and ccleaner did nothing either :/), I had to re-enable win10 mode with editing the registry entry: HKEY_CURRENT_USER\Software\Microsoft\Windows\CurrentVersion\Explorer\ExplorerPatcher

**Reply by MikuDroid** [reply]
thanks, i can't find properties menu under win + x. changing registry entry works for enabling win10 taskbar.

**Reply by valinet** [reply]
rundll32 C:\Windows\dxgi.dll,ZZGUI
